eng/pipelines/coreclr/templates/build-perf-maui-apps-net7.yml

```diff
@@ -112,17 +112,18 @@ steps:
 
   # Get the current maui nuget config so all things can be found and darc based package sources are kept up to date.
   - script: |
-      echo '' > ./global.json 
+      echo '{}' > ./global.json 
     displayName: Create global.json
     workingDirectory: $(Build.SourcesDirectory)
   
   - script: |
       curl -o NuGet.config 'https://raw.githubusercontent.com/dotnet/maui/main/NuGet.config'
       curl -o dotnet-install.sh 'https://dotnet.microsoft.com/download/dotnet/scripts/v1/dotnet-install.sh'
       chmod -R a+rx .
-      ./dotnet-install.sh --version 7.0.100-preview.6.22316.8 --install-dir .
+      cp $(Build.SourcesDirectory)/src/tests/Common/maui/rollbackfile-net7.json ./rollbackfile-net7.json
+      ./dotnet-install.sh --version 7.0.100-preview.7.22376.5 --install-dir .
       ./dotnet --info
-      ./dotnet workload install maui --from-rollback-file https://aka.ms/dotnet/maui/net7.0.json --configfile NuGet.config
+      ./dotnet workload install maui --from-rollback-file rollbackfile-net7.json --configfile NuGet.config
```

Comment:
I think having a local file is the best for now. In the next couple weeks, we should be able to use https://aka.ms/dotnet/maui/net7.0.json or some other URL again.